bentoml/_internal/utils/__init__.py

```diff
@@ -211,9 +211,20 @@ def copy_file_to_fs_folder(
     dst_folder_path: str = ".",
     dst_filename: t.Optional[str] = None,
 ):
-    """Copy the given file at src_path to dst_fs filesystem, under its dst_folder_path
-    folder with dst_filename as file name. When dst_filename is None, keep the original
+    """
```

Comment:
probably no need to change this at all - i'd prefer not using `:code:` for internal docs and Args section doesn't really add any value

Reply:
it actually helps with lsp hover